lib/oneview-sdk/resource.rb

```diff
@@ -361,6 +361,12 @@ def recursive_like?(other, data = @data)
         return false unless data && data.respond_to?(:[])
         if val.is_a?(Hash)
           return false unless data.class == Hash && recursive_like?(val, data[key.to_s])
+        elsif val.is_a?(Array) && val.first.is_a?(Hash)
```

Comment:
I'm OK with this change as long as we can verify that it won't negatively impact other resources, but the only way I can think of verifying that is by looking through every single resource's API docs. I think I'd rather update this code than do that. lol